tests/integration_tests/bugs/test_lp1813396.py

```diff
@@ -0,0 +1,33 @@
+"""Integration test for lp-1813396
+
+Ensure gpg works even if VM provides no /dev/tty"""
+
+import pytest
+
+from tests.integration_tests.instances import IntegrationInstance
+from tests.integration_tests.log_utils import ordered_items_in_text
+
+
+USER_DATA = """\
+#cloud-config
+apt:
+  sources:
+    docker:
+      source: 'deb [arch=amd64] https://download.docker.com/linux/debian stretch stable'
```

Comment:
Are we better using a PPA for this? Easier to access when we're running tests inside our network, and we don't introduce a dependency on an "external" service.